pkg/prowgen/prowgen_config.go

```diff
@@ -353,6 +357,81 @@ func NewGenerateConfigs(ctx context.Context, r Repository, cc CommonConfig, opts
 	return cfgs, nil
 }
 
+// NewProwConfig returns a prow config for branch protection and tide config like https://github.com/openshift/release/blob/363307d181d1cf4734191bb794be20df54431b7d/core-services/prow/02_config/openshift-knative/eventing-integrations/_prowconfig.yaml
+func NewProwConfig(r Repository) shardprowconfig.ProwConfigWithPointers {
+	tideMissingLabels := []string{
+		"backports/unvalidated-commits",
+		"do-not-merge/hold",
+		"do-not-merge/invalid-owners-file",
+		"do-not-merge/work-in-progress",
+		"jira/invalid-bug",
+		"needs-rebase",
+	}
+	return shardprowconfig.ProwConfigWithPointers{
+		BranchProtection: &config.BranchProtection{
+			Orgs: map[string]config.Org{
+				r.Org: config.Org{
+					Repos: map[string]config.Repo{
+						r.Repo: config.Repo{
+							Branches: map[string]config.Branch{
+								"release-next": config.Branch{
+									Policy: config.Policy{
+										Protect: ptr.To(false),
+									},
+								},
+								"release-next-ci": config.Branch{
+									Policy: config.Policy{
+										Protect: ptr.To(false),
+									},
+								},
```

Comment:
The release-next branches are getting recreated constantly. Will this not break it?

Reply:
Just noticed, the value is `false`. Closing. Sorry for the noise!